### PR TITLE
Add ci tutorial test

### PR DESCRIPTION
**Pull request by giusebar**
Add script to test the snap from developer guide part 2 more thoroughly.

### CODE REVIEW

**Review by artivis — APPROVED** [review]
lgtm.
However relying on `sleep`s may results on flaky tests every now and then.

**Review by Guillaumebeuzeboc — APPROVED** [review]

File: .github/workflows/snap.yaml [hunk 1]
```diff
@@ -45,32 +45,42 @@ jobs:
...
+                            local current_time=$(date +%s)
+                            local elapsed_time=$((current_time - start_time))
+
+                            if [ $elapsed_time -ge $timeout ]; then
+                                echo "Error $name is not running."
+                                sudo snap stop turtlebot3c
+                                killall -9 rosmaster
+                                exit 1
+                            fi
+                            sleep 1
```
Comment (Guillaumebeuzeboc) [inline .github/workflows/snap.yaml:69]: Why do you need a sleep here?

Comment (giusebar) [reply .github/workflows/snap.yaml:69]: just for the while loop

**Review by Guillaumebeuzeboc — APPROVED** [review]

**Review by Guillaumebeuzeboc — APPROVED** [review]

**Review by MirkoFerrati — APPROVED** [review]
reviewed together over video call

**Review by artivis — APPROVED** [review]

**Review by Guillaumebeuzeboc — APPROVED** [review]

**Review by Guillaumebeuzeboc — APPROVED** [review]

**Pull request merged**